src/settings/BatchGroupsSettings/BatchGroupsSettings.js

```diff
@@ -1,7 +1,62 @@
 import React from 'react';
+import {
+  FormattedMessage,
+  injectIntl,
+  intlShape,
+} from 'react-intl';
 
-const BatchGroupsSettings = () => {
-  return <div data-test-batch-groups-settings />;
+import { stripesShape } from '@folio/stripes/core';
+import { ControlledVocab } from '@folio/stripes/smart-components';
+
+const columnMapping = {
+  name: <FormattedMessage id="ui-invoice.settings.batchGroups.column.name" />,
+  description: <FormattedMessage id="ui-invoice.settings.batchGroups.column.description" />,
+};
+const SYSTEM_GROUP_ID = '2a2cb998-1437-41d1-88ad-01930aaeadd5';
+
+class BatchGroupsSettings extends React.Component {
+  constructor(props) {
+    super(props);
+    this.connectedControlledVocab = props.stripes.connect(ControlledVocab);
+  }
+
+  render() {
+    const { intl, stripes } = this.props;
+    const hasPerm = stripes.hasPerm('settings.invoice.enabled');
+    const getDisableAttr = () => ({
```

Comment:
is it really required? user shouldn't have possibility to open page without `settings.invoice.enabled` perm

Reply:
@NikitaSedyx you're right, it could be much simpler, please check this out